dags/pipeline_small.py

```diff
@@ -99,6 +100,21 @@ def remove_unblurred_images(**context):
         template_searchpath=["/"],
         catchup=False,
 ) as dag:
+    @task
+    def get_config(**context):
+        workers = context['dag_run'].conf.get('workers')
+        if workers:
+            if workers.isdigit() and (0 < int(workers) < 10):
+                print(f"Processing using {workers} workers.")
+                return int(workers)
+            else:
+                print("Default to 1 worker.")
+                return 1
+        else:
+            print("Default to 1 worker.")
+            return 1
+
```

Comment:
Je zou beide `else` clauses weg kunnen laten, en gewoon `1` terug kunnen geven wanneer de er niet voldaan wordt aan de condities in de `if` statement